submitter/relayer/relayer.go

```diff
@@ -80,9 +80,15 @@ func (rl *Relayer) SendCheckpointToBTC(ckpt *ckpttypes.RawCheckpointWithMetaResp
 		return nil
 	}
 
-	if rl.lastSubmittedCheckpoint == nil || rl.lastSubmittedCheckpoint.Epoch < ckptEpoch {
+	if rl.lastSubmittedCheckpoint == nil ||
+		rl.lastSubmittedCheckpoint.Tx1 == nil ||
+		rl.lastSubmittedCheckpoint.Epoch < ckptEpoch {
 		rl.logger.Infof("Submitting a raw checkpoint for epoch %v for the first time", ckptEpoch)
 
+		if rl.lastSubmittedCheckpoint == nil {
+			rl.lastSubmittedCheckpoint = &types.CheckpointInfo{}
```

Comment:
Init the `lastSubmittedCheckpoint`

Reply:
Can we add this to the default structure at `New( ` https://github.com/babylonlabs-io/vigilante/blob/c8a5c3638e13a024676b143dd84b8de33041433f/submitter/relayer/relayer.go#L47

and then simplify the check by removing `rl.lastSubmittedCheckpoint == nil`